packages/desktop-client/src/components/reports/Charts.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+
+import * as monthUtils from 'loot-core/src/shared/months';
+import { integerToCurrency } from 'loot-core/src/shared/util';
+import { View, Text, Block, Button } from 'loot-design/src/components/common';
+import { colors, styles } from 'loot-design/src/style';
+
+const fontWeight = 600;
+
+export function ChartItem({
+  Icon,
+  title,
+  style,
+  handleClick,
+  bold,
+  header,
+  id,
+  reportPage,
+}) {
+  const hoverStyle = {
+    color: colors.b4,
+    cursor: 'pointer',
+    backgroundColor: 'inherit',
+  };
+  const linkStyle = [
+    header && styles.largeText,
+    {
+      color: colors.n7,
+      fontWeight: bold ? fontWeight : null,
+    },
+    { ':hover': hoverStyle },
+  ];
+
+  const content = (
+    <View
+      style={{
+        flexDirection: 'row',
+        alignItems: 'center',
+        height: 16,
+        color: reportPage === id ? colors.n3 : 'inherit',
+      }}
+    >
+      {Icon && (
+        <Icon id={id} width={12} height={12} style={{ color: 'inherit' }} />
+      )}
+      <Block id={id} style={{ marginLeft: Icon ? 3 : 0, color: 'inherit' }}>
+        {title}
+      </Block>
+    </View>
+  );
+
+  return (
+    <View style={[{ flexShrink: 0 }, style]}>
+      <Button
+        id={title + 'Button'}
```

Comment:
I’m noticing there are a bunch of `id` props. Are they necessary? I don’t think many other parts of the codebase use `id`.

Reply:
Removed